lib/lightning/config/bootstrap.ex

```diff
@@ -99,7 +99,7 @@ defmodule Lightning.Config.Bootstrap do
           :string,
           Utils.get_env([:lightning, :apollo, :endpoint])
         ),
-      openai_api_key: env!("OPENAI_API_KEY", :string, nil)
+      openai_api_key: env!("ANTHROPIC_API_KEY", :string, nil)
```

Comment:
should the key name here be more generic? maybe change `openai_api_key` to `apollo_3rd_party_key`? or am i misunderstanding this change?

Reply:
That's a good question.

It certainly isn't a generic apollo API key. Different apollo services require different API keys.

It might make sense to rename to `AI_ASSISTANT_API_KEY` though. That way we're binding it to the service, rather than the implementation. Of course it's also less declarative from a devops point of view.